test/bootleg/tasks/build_task_functional_test.exs

```diff
@@ -38,6 +38,18 @@ defmodule Bootleg.Tasks.BuildTaskFunctionalTest do
     end)
   end
 
+  test "builds the application with a release_workspace path", %{hosts: [host], project_location: location} do
+    use Bootleg.Config
+
+    role :build, host.ip, release_workspace: "/home/#{host.user}/release_workspace", port: host.port
+
+    File.cd!(location, fn ->
+      capture_io(fn ->
+        invoke :build
```

Comment:
Thank you for the tests! I would feel more comfortable if there was an assertion here, maybe we can have an assertion that the tgz file exists in the remote workspace. - I also noticed that the test this one was based off of also doesn't actually have an assertion, i'll file an issue for that one

assertion can probably be something along the lines of:
```
...
invoke :build
assert [{:ok, ["#{Config.app()}.tar.gz"], 0, _}] = remote :build, "ls /home/#{host.user}/release_workspace"
...
```